matching-game/jest.config.mjs

```diff
@@ -0,0 +1,19 @@
+import nextJest from 'next/jest.js'
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: './'
+})
+
+// Add any custom config to be passed to Jest
+/** @type {import('jest').Config} */
+const config = {
+  // Add more setup options before each test is run
+  // setupFilesAfterEnv: ['<rootDir>/jest.setup.ts'],
+
+  testEnvironment: 'jest-environment-jsdom',
+  setupFilesAfterEnv: ['./jest.setup.ts']
+}
+
+// createJestConfig is exported this way to ensure that next/jest can load the Next.js config which is async
+export default createJestConfig(config)
```

Comment:
ignore most of this. it's mostly copied from Next.js documentation for setting up Jest.